assignments/prototypes.js

```diff
@@ -15,6 +17,25 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.dimensions = attributes.dimensions;
+  this.destroy = function () {
+    return 'Object was removed from the game.';
```

Comment:
How can we make it so that it returns that the particular name of the game object was removed from the game?